drivers/include/saul/periph.h

```diff
@@ -19,19 +19,19 @@
 #ifndef SAUL_PERIPH_H
 #define SAUL_PERIPH_H
 
-#ifdef MODULE_SAUL_GPIO
+#if MODULE_SAUL_GPIO || DOXYGEN
 #include "periph/gpio.h"
 #endif /* MODULE_SAUL_GPIO */
 
-#ifdef MODULE_SAUL_ADC
+#if MODULE_SAUL_ADC || DOXYGEN
 #include "periph/adc.h"
 #endif /* MODULE_SAUL_ADC */
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-#ifdef MODULE_SAUL_GPIO
+#if MODULE_SAUL_GPIO || DOXYGEN
 typedef enum {
```

Comment:
now that doxygen gets here, this `enum` needs a proper Doxygen comment, i.e., something with `@brief` (see struct below for instance), could you add please add one.

Reply:
please amend directly.